src/getInstallArgs.ts

```diff
@@ -0,0 +1,10 @@
+import { InstallCmd } from './getInstallCmd';
+
+export default function getInstallArgs(cmd: InstallCmd, packages: string[]) {
+  switch (cmd) {
+    case 'npm':
+      return ['install', ...packages, '--save-dev'];
```

Comment:
Using `switch` syntax on `"yarn" | "npm"` here to ensure the return type is never `undefined`